docs/daemon/configuration/genesis.mdx

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 2
+sidebar_label: "Genesis"
+id: daemon-genesis
+title: Genesis
+description: Genesis file configuration
+slug: /daemon/genesis
+---
+Genesis file in `rootdir/abci/config/` defines the initial kwild state upon the initialization of the kwild network
+
+- `genesis_time`: This field specifies the official time at which the blockchain is initialized.
```

Comment:
Breaking up this page with subsections can definitely help readability.  I think it should be structured as follows:

`Intro`: Covers what is a genesis file. You should also include the shortest possible valid genesis file as an example
`Setup`: How to set up a genesis file from scratch
`Settings`: A list of the different settings that can be set (which you already have)
`Full Example`: A full example of a genesis file with everything configured.